package.json

```diff
@@ -336,7 +336,7 @@
     "re-resizable": "^6.1.1",
     "re2": "^1.15.4",
     "react": "^16.12.0",
-    "react-ace": "^5.9.0",
+    "react-ace": "^7.0.5",
```

Comment:
@timroes Any suggestions on the best way to verify this hasn't broken anything?

Reply:
I don't think we have any better way besides manually testing a couple of places we use the editor. I'd suggest looking into at least one read only case, one write case, and advanced settings (which I think is the only place using dynamic heights on EuiCodeEditor).

Reply:
Thanks Tim! Tested Advanced Settings to cover both the write and dynamic heights use cases, and tested the Rollup Jobs JSON tab for the read-only case. All seems to behave as expected.